Sources/Gatekeeper/GatekeeperMiddleware.swift

```diff
@@ -0,0 +1,24 @@
+import Vapor
+
+public struct GatekeeperMiddleware {
+    let gatekeeper: Gatekeeper
+}
+
+extension GatekeeperMiddleware: Middleware {
+    public func respond(
+        to request: Request,
+        chainingTo next: Responder
+    ) throws -> EventLoopFuture<Response> {
```

Comment:
`Future`

Reply:
Will be replaced with EventLoopFuture anyways with next major Vapor release AFAIK.

Reply:
yeah you might be right about that